eclass/qt5-build.eclass

```diff
@@ -631,7 +631,7 @@ qt5_base_configure() {
 		-no-compile-examples
 
 		# disable rpath on non-prefix (bugs 380415 and 417169)
-		$(usex prefix '' -no-rpath)
+		$(if in_iuse prefix && use prefix; then echo ''; else echo -no-rpath; fi)
```

Comment:
which can be shortened to `{ in_iuse prefix && use prefix; } || echo -no-rpath`, right?